receiver/awss3receiver/README.md

```diff
@@ -46,4 +48,23 @@ receivers:
         s3_bucket: "mybucket"
         s3_prefix: "trace"
         s3_partition: "minute"
-```
\ No newline at end of file
+```
+
+## Notifications
+The receiver can send notifications of ingest progress to an OpAmp server using the custom message capability of 
+"org.opentelemetry.collector.receiver.awss3" and message type "TimeBasedIngestStatus".
+The format of the notifications is a JSON object with the following fields:
+
+| Field             | Description                                                                     |
+|:------------------|:--------------------------------------------------------------------------------|
+| `telemetry_type`  | The type of telemetry being ingested. One of "traces", "metrics", or "logs".    |
+| `ingest_status`   | The status of the data ingestion. One of "ingesting", "failed", or "completed". |
+| `start_time`      | The time to start retrieving data in RFC3339 format.                            |
+| `stop_time`       | The time to stop retrieving data in RFC3339 format.                             |
+| `ingest_time`     | The time of the data currently being ingested in RFC3339 format.                |
+| `failure_message` | Error message if `ingest_status` is "failed".                                   |
```

Comment:
Is there a spec for this as part of opAmp or maybe entities?

Reply:
There is no spec for this and that is the reason why I'm using a Custom Capability.